CMakeLists.txt

```diff
@@ -12,15 +12,12 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-cmake_minimum_required(VERSION 2.8.7)
-
-project(trower-base64)
```

Comment:
The Makefies without the changes do not work for RDK-B builds, specifically
the ARM build of the XB3 platforms.

These changes allow this GitHub repo to be used for RDK-B community builds


On Wednesday, 15 June 2016, Weston Schmidt <notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/Comcast/trower-base64/pull/3#discussion_r67116582>:
>
> > @@ -12,15 +12,12 @@
> >  #   See the License for the specific language governing permissions and
> >  #   limitations under the License.
> >
> > -cmake_minimum_required(VERSION 2.8.7)
> > -
> > -project(trower-base64)
>
> Why rename the project?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Comcast/trower-base64/pull/3/files/9faba35d3a55d0726740beac432d93545b2a965e#r67116582>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGC8oJuXp0BWn2BDOl1hwF9xx0Mo9T4Aks5qL7J0gaJpZM4ItUEP>
> .
>